src/chfn.c

```diff
@@ -643,6 +644,10 @@ int main (int argc, char **argv)
 	 * name, or the name getlogin() returns.
 	 */
 	if (optind < argc) {
+		if (!is_valid_user_name (argv[optind])) {
```

Comment:
I used the same convention as in the current files. For changing formatting of the code I would suggest a separate PR and using clang format functionality, instead of manual formatting.